src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -1,54 +1,55 @@
 ---
-title: "Configuration&#58 Docker"
+title: "Configuring Che on Docker"
 keywords: docker, configuration
 tags: [installation, docker]
 sidebar: user_sidebar
 permalink: docker-config.html
 folder: setup-docker
 ---
 
+The Che on Docker configuration is handled by modifying the https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env[che.env] file that is placed in the root directory of a host folder volume mounted to `:/data`. This configuration file is generated during the `che init` phase. If you rerun the `che init` command in an already initialized folder, the process will abort unless you pass `--force`, `--pull`, or `--reinit`.
```

Comment:
The `che.env` file linked to here should also be styled as it is throughout the rest of the documentation.